dabplus-on-air-processing.conf

```diff
@@ -5,7 +5,11 @@
 
 # configuration for processor
 #
-enable_processor = true
+# Choose which processor to use, possible choices are:
+# * internal - simple processor inspired by conduit
```

Comment:
"internal" ist in this repo, mk is not... I'll rename it to simple or something else when I add the next one and "internal" isn't descriptive enough anymore.

Feel free to hack away if it bugs you too much ;)